_resources/port1.0/group/crossgcc-1.0.tcl

```diff
@@ -49,6 +49,12 @@ options crossgcc.target \
 
 default crossgcc.languages {{c c++}}
 
+# TODO: Can we use a table like this to automatically set the checksums for the port?
+set crossgcc.versions_info {
+    7.1.0  bzip2  a228dc45a09eda91b1a201d234f9013b3009b461  8a8136c235f64c6fef69cac0d73a46a1a09bb250776a050aec8f9fc880bebc17
+    7.2.0  xz     fa8eed36c78cf135f9cc88e60845996b5cfaba52  1cf7adf8ff4b5aa49041c8734bbcf1ad18cc4c94d0029aae0f4e48841088479a
+}
```

Comment:
I like the idea. I think we could safely add the checksums to the port group, even if ports are still overriding it later. Dropping checksums or converting to checksums-append in the ports can be done afterwards.

Reply:
Are you willing to help me with hints about how to properly do that? I gladly add the checksums, but I'm not sure how to use them. I use a similar trick for geant4, but I create ports on the fly there. I would probably need some kind of a "hash table" to store the checksums here. Or maybe loop through all the entries and just test whether any of them matches?